src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,265 @@
+import json
+
+from py42.clients import BaseClient
+from py42.clients.util import get_all_pages
+
+
+class HighRiskEmployeeClient(BaseClient):
+    """
+    Administrator utility to manage High Risk employees.
+    `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
+
+    """
+
+    # In other modules URI is a single variable, breaking it up to have better control
```

Comment:
You can remove this comment, it is fine either way. We don't have to refactor anything right now. It may be a good idea for some APIs. Others, like this one, have changed dramatically between v1 and v2.

Reply:
Removed.